doc/dev/msgr2.rst

```diff
@@ -838,3 +838,56 @@ _____________________________________
                 |                   |
 
 
+.. graphviz::
```

Comment:
!! That's so much easier than what I have here

https://github.com/ceph/ceph/blob/ea706273b53ea25232982fc485cbbf056f7203b6/doc/cephfs/mds-state-diagram.dot

Including a Makefile and svg in git. I'll have to make a note to migrate that into the rst.

Reply:
https://tracker.ceph.com/issues/50865

Reply:
@batrick sphinx also allows us to include a dot file by referencing it. see https://www.sphinx-doc.org/en/master/usage/extensions/graphviz.html. i think the upside of using a dot file is that it's easier to be process with external tools.